js/gradient-generator.js

```diff
@@ -1,27 +1,25 @@
 //settings
-const fps = 260;
-const duration = 3;
-let firstGrad = true;
+const fps = 260; // is there a better name for this?
+const duration = 3; // duration of what?
+const firstGrad = true; // what does this mean? 
 //create set of arrays of random numbers
-const createRandomGradient = () => {
-	let gradient = {
+const createRandomGradient = () => ({ // return object literal
 		rgbOne: [createColor(),createColor(),createColor()],
 		rgbTwo: [createColor(),createColor(),createColor()]
-	};
-	return gradient;
-}
-const startTransition = (onLoad) => {
-	let targetColor = createRandomGradient();
-	targetColor1 = targetColor.rgbOne;
-	targetColor2 = targetColor.rgbTwo;
+});
+
+const startTransition = hasLoaded => { // if it's a boolean, hasLoaded or isLoaded than onLoad, because onLoad sounds like a function
+	const targetColor = createRandomGradient();
+	targetColorOne = targetColor.rgbOne;
+	targetColorOne = targetColor.rgbTwo;
 	window.transitionHandler = setInterval( () => {
-		transitionGradient(onLoad);
+		transitionGradient(hasLoaded);
```

Comment:
"onLoad" is supposed to indicate the fact that it happens on page load not by user interaction. That is why I thought "onLoad" would be better.